docs/howto/features/ephemeral.md

```diff
@@ -170,8 +201,67 @@ jupyterhub:
           url: ""
 ```
 
+## Use `nbgitpuller` for distributing content
+
+We encourage users to use [nbgitpuller](https://github.com/jupyterhub/nbgitpuller)
+for distributing content. This allows creation of a specific link that will
+put users who click it on a specific notebook with a specific UI (such as lab,
+classic notebook, RStudio, etc).
+
+The [nbgitpuller link generator](http://nbgitpuller.link/) supports mybinder.org
+style links, but for use with *ephemeral hubs*, just use the regular 'JupyterHub'
+link generator. [Firefox](https://addons.mozilla.org/en-US/firefox/addon/nbgitpuller-link-generator/)
+and [Google Chrome](https://chrome.google.com/webstore/detail/nbgitpuller-link-generato/hpdbdpklpmppnoibabdkkhnfhkkehgnc)
+extensions are also available.
+
+## Enable stronger anti-crypto abuse features for a hub
+
+Ephemeral hubs with public access must have [stronger anti crypto abuse features
+enabled](howto:features:cryptnono) before going live.
+
+## (Optional) Shared systemwide password setup
+
+Optionally, in addition to the cryptnono setup documented above, a shared
+password may be used as additional protection to guard against random cryptobros
+abusing using our systems for 'free' compute. This password is same for all
+users, and can be shared by the community champions of the hub in whatever form
+they wish.
+
+This requires two bits of config - one in the `<hub>.values.yaml` file for the hub,
+and another in the `enc-<hub>.secret.values.yaml` file.
+
+In `<hub>.values.yaml`:
+
+```yaml
+ingressBasicAuth:
+  enabled: true
+
+jupyterhub:
+  ingress:
+    annotations:
+      # We protect our entire hub from cryptobros by putting it all
+      # behind a single shared basicauth
```

Comment:
```suggestion
      # We protect our entire hub from cryptominers by putting it all
      # behind a single shared basicauth
```

Its subjective of course, but I appreciate a more neutral language - wdyt?

Reply:
Ah, this is a rabbithole that I'd prefer to not get into in this PR. Is it alright if we just let this one be for now?